server/models/User.js

```diff
@@ -90,7 +90,18 @@ export default (Sequelize, DataTypes) => {
     description: DataTypes.STRING,
     longDescription: DataTypes.TEXT,
     isOrganization: DataTypes.BOOLEAN, // e.g. DigitalOcean, PubNub, ...
-
+    isHost: DataTypes.BOOLEAN, // e.g. WWCode
```

Comment:
Personally, I think that creates more confusion over time - especially as we are often manually manipulating the database. So, now when Pia creates a host, she'll have to flip additional bits and forgetting it will create inconsistency.